json-schemas/definitions.json

```diff
@@ -14,6 +14,14 @@
     "did": {
       "type": "string",
       "pattern": "^did:([a-z0-9]+):((?:(?:[a-zA-Z0-9._-]|(?:%[0-9a-fA-F]{2}))*:)*((?:[a-zA-Z0-9._-]|(?:%[0-9a-fA-F]{2}))+))((;[a-zA-Z0-9_.:%-]+=[a-zA-Z0-9_.:%-]*)*)(\/[^#?]*)?([?][^#]*)?(#.*)?$"
+    },
+    "dateCreated": {
```

Comment:
Sanity: Not seeing any reference to this definition. Expecting something like [this](https://github.com/TBD54566975/dwn-sdk-js/blob/ecd72c95e4ab31f68dbd74d36bc3502967d163da/json-schemas/collections/collections-write.json#L24)